app/models/immunisation_import.rb

```diff
@@ -175,6 +202,30 @@ def recorded_at
       Time.zone.now
     end
 
+    def patient_first_name
+      @data["PERSON_FORENAME"]
+    end
+
+    def patient_last_name
+      @data["PERSON_SURNAME"]
+    end
+
+    def patient_date_of_birth
+      @data["PERSON_DOB"]
+    end
+
+    def patient_nhs_number
+      @data["NHS_NUMBER"]
+    end
+
+    def school_name
+      @data["SCHOOL_NAME"]&.strip
```

Comment:
Should we be calling `strip` on all string values if we need it here?

Reply:
Possibly... I'm mainly doing it here in an attempt to clean up the data as the example CSV contains spaces

<img width="462" alt="Screenshot 2024-07-18 at 09 19 14" src="https://github.com/user-attachments/assets/af877c16-3442-45bc-b227-840dc55facd1">

Reply:
If random whitespace can appear in the examples, I'd be tempted to apply it to all our string fields just in case.

Reply:
Yeah, I'll push something up